docs/release-notes/release-notes-0.14.2.md

```diff
@@ -32,6 +32,13 @@
   allow increasing or decreasing the balance of an off-chain account by a
   specified amount.
 
+
+* The [`accounts update` `--new_balance` flag has been
+  deprecated](https://github.com/lightninglabs/lightning-terminal/pull/974).
+  Use the `accounts credit` and `accounts debit` commands instead, to update
+  an account's balance. The flag will no longer be supported in Lightning
+  Terminal `v0.15.0-alpha`.
```

Comment:
Realized that we should push this to `v0.16.0`, as lnd `v0.19.0` will be `v0.15.0`. I'll update this a bit later today.

Reply:
still unaddressed i think 🙏